core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -145,6 +145,7 @@ registry.ci.openshift.org/origin/4.11:hyperkube quay.io/openshift/origin-hyperku
 registry.ci.openshift.org/origin/4.11:hypershift quay.io/openshift/origin-hypershift:4.11 quay.io/openshift/origin-hypershift:4.11.0 quay.io/openshift/origin-hypershift:latest
 registry.ci.openshift.org/origin/4.11:hypershift-operator quay.io/openshift/origin-hypershift-operator:4.11 quay.io/openshift/origin-hypershift-operator:4.11.0 quay.io/openshift/origin-hypershift-operator:latest
 registry.ci.openshift.org/origin/4.11:ibm-cloud-controller-manager quay.io/openshift/origin-ibm-cloud-controller-manager:4.11 quay.io/openshift/origin-ibm-cloud-controller-manager:4.11.0 quay.io/openshift/origin-ibm-cloud-controller-manager:latest
+registry.ci.openshift.org/origin/4.11:ibm-powervs-cloud-controller-manager quay.io/openshift/origin-ibm-powervs-cloud-controller-manager:4.11 quay.io/openshift/origin-ibm-powervs-cloud-controller-manager:4.11.0 quay.io/openshift/origin-ibm-powervs-cloud-controller-manager:latest
```

Comment:
elsewhere we have "powervs" as the prefix for this platform (`origin-powervs-machine-controllers`).

we should be consistent between the two unless something distinct is actually intended by `ibm-powervs` vs `powervs`

Reply:
(my sense is that this new image should be `powervs-cloud-controller-manger`)

Reply:
Updated the image name to `powervs-cloud-controller-manager`